### DISCUSSION TITLE
Would there be any way to add protection against other thrown objects, such as axes?

### DISCUSSION THREAD

**Discussion by VirtualizedTrash** [GriefPrevention/GriefPrevention#1674] [Q&A]
I run a datapack that allows players to throw swords and axes. It's relatively bug free, but there's a massive problem. Swords and axes can kill mobs in claims, and also break blocks in claims. Is there any way to fix that through the plugin, or would I have to modify the datapack instead? 

https://www.planetminecraft.com/data-pack/throwable-swords/

**Comment by Jikoo** [comment]
In general datapacks will never play well with plugins. They don't do things in a way that fires Bukkit events for equivalent actions (and can't, unfortunately). Consider finding a plugin that provides the same features - not only will it be more efficient, it will be much more capable of supporting other plugins.
